### PR TITLE
Update mdn_urls: remove 404s, replace redirects, add new pages

### PR DESCRIPTION
**Pull request by Elchi3**
This PR is the initial result of automatically updating mdn_urls in https://github.com/mdn/browser-compat-data/pull/23431.

### CODE REVIEW

File: javascript/operators/conditional.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
       "conditional": {
         "__compat": {
           "description": "Conditional operator (<code>c ? t : f</code>)",
-          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Conditional_Operator",
```
Comment (Josh-Cena) [inline javascript/operators/conditional.json:7]: This differs from the real URL (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) by capitalization only

File: javascript/operators/import.json [hunk 2]
```diff
@@ -3,7 +3,6 @@
     "operators": {
       "import": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Import",
```
Comment (Josh-Cena) [inline javascript/operators/import.json:6]: Same here

File: javascript/operators/property_accessors.json [hunk 3]
```diff
@@ -4,7 +4,6 @@
       "property_accessors": {
         "__compat": {
           "description": "Property accessors",
-          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Property_Accessors",
```
Comment (Josh-Cena) [inline javascript/operators/property_accessors.json:7]: Same here

**Review by Josh-Cena — COMMENTED** [review]
The script doesn't seem able to deal with capitalization differences. They should probably be fixed too

**Comment by Josh-Cena** [comment]
FWIW this seems slightly inconvenient in the context of fixing MDN gaps, since every new doc now also requires a BCD update, but I think it's generally an improvement, and it'll probably take us a long while to fix all MDN gaps...

File: webextensions/api/scripting.json [hunk 4]
```diff
@@ -3,7 +3,6 @@
     "api": {
       "scripting": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/",
```
Comment (rebloor) [inline webextensions/api/scripting.json:6]: @Elchi3 this link is valid, no redirect that I can see

Comment (Elchi3) [reply webextensions/api/scripting.json:6]: Yeah, there was a trailing slash, however.

**Comment by Elchi3** [comment]
Thanks @Josh-Cena and @rebloor!

> The script doesn't seem able to deal with capitalization differences. They should probably be fixed too

Fixed!

**Review by queengooborg — APPROVED** [review]
A quick spot check shows favorable  results, so LGTM!

**Pull request merged**